app/views/search/index.api.rsb

```diff
@@ -3,13 +3,16 @@ api.array :results, api_meta(:total_count => @result_set.n_hits,
                              :limit => @search_request.limit) do
   @result_set.each do |record|
     api.result do
-      api.id          record.event_id
-      api.title       record.event_highlighted_title
-      api.type        record.event_type
-      api.url         url_for(record.event_url(:only_path => false))
-      api.description record.event_content_snippets.join("\n")
-      api.datetime    record.event_datetime
-      api.rank        record.rank
+      api.id               record.event_id
+      api.title            record.event_highlighted_title
+      api.type             record.event_type
+      api.url              url_for(record.event_url(:only_path => false))
+      api.description      record.event_content_snippets.join("\n")
+      # TODO: 'datetime' is deprecated but maintained for backward compatibility. Use 'last_modified_at' or 'registered_at' explicitly.
```

Comment:
Could you remove `TODO`?
Could you fold a long line?

```suggestion
      # 'datetime' is deprecated but maintained for backward compatibility.
      # Use 'last_modified_at' or 'registered_at' explicitly.
```

Reply:
fix: [846e1ac](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/846e1acdcadab3275950a243c7081aa569b3cc78)